blocks/library/image/index.js

```diff
@@ -82,40 +85,50 @@ registerBlockType( 'core/image', {
 
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, alt, caption } = attributes;
+		const updateAlt = ( newAlt ) => setAttributes( { alt: newAlt } );
+
+		const inspector = focus && (
+			<InspectorControls key="inspector">
+				<TextControl label={ __( 'Alternate Text' ) } value={ alt } onChange={ updateAlt } />
```

Comment:
I'm afraid if we come up with something like that, it would add a new thing to understand.